src/components/ArticleList.jsx

```diff
@@ -0,0 +1,37 @@
+import axios from "axios";
+import { useEffect, useState } from "react";
+
+const ArticleList = () => {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    axios
+      .get("https://iz-nc-news.onrender.com/api/articles")
```

Comment:
For the sake of readability, I would definitely suggest to pull out anything to do with axios into a separate utils folder/file - then you can utilize an axios.create and separate out the concerns into separate functions you can then import!